### DISCUSSION TITLE
mounting up issue

### DISCUSSION THREAD

**Discussion by NightrunLP** [mod-playerbots/mod-playerbots#794] [General]
I hit level 68 with a new group of 5 characters (me and 4 of my alts).
When I mount up on a flying mount, all of them do. When I mount up on a ground mount, only two of them will follow my lead - two bots will stay on foot no matter what I do.
Has anybody experienced this and found a solution?

**Comment by Bobblybook** [comment]
I wonder if this is related to something I was seeing - there are some mounts that are classified as scaling ground/flying mounts and they seem to bug out the bots when trying to mount-match the player. I think I blacklisted the problematic mounts I found, but there may be more.

I've implemented a custom table where you can specify mounts used for each character, that helped me get around the issue.
You could possibly try and report back if it solves your issue. It doesn't yet have a command to add/remove mounts and it needs to be set manually in the db.

data/sql/playerbots/base/playerbots_preferred_mounts.sql

In the db, add an entry to this table with the guid of the character, spellid of the mount, and type (0=ground, 1=fly).

**Comment by Bobblybook** [comment]
Also I believe this may also be due to your alts having or not having flying skill, have you checked that?

**Comment by NightrunLP** [comment]
Hi, thanks for the quick reply :)
They do all have the flying skill, logged on each and everyone of them to buy it seperately.
I'll check your table during the weekend, it sounds like a good solution! :)
